### PR TITLE
[DOCFIX] Doc fix for Chinese version for multiple file

### PR DESCRIPTION
**Pull request by Popeye13**

### CODE REVIEW

**Comment by ifcharming** [comment]
Hi @Popeye13 thanks for the PR!
Mostly looks good, can you please resolve the conflicts?

**Comment by ifcharming** [comment]
Hi @Popeye13 , have you got a chance to resolve the merge conflicts in this PR? thanks

File: docs/cn/index.md [hunk 1]
```diff
@@ -24,10 +24,17 @@ Alluxio为大数据软件栈带来了显著的性能提升。例如，[百度](h
 
 Alluxio与Hadoop是兼容的。现有的数据分析应用，如Spark和MapReduce程序，可以不修改代码直接在Alluxio上运行。Alluxio是一个已在多家公司部署的开源项目([Apache License 2.0](https://github.com/alluxio/alluxio/blob/master/LICENSE))。
 Alluxio是发展最快的开源大数据项目之一。自2013年4月开源以来，已有超过100个组织机构的
+<<<<<<< HEAD
```
Comment (apc999) [inline docs/cn/index.md]: you still have merge conflicts here. 
you should remove line 27 to 32, and the line 37

**Comment by ifcharming** [comment]
Jenkins, test this please

**Comment by ifcharming** [comment]
LGTM

**Pull request merged**